src/engine/Algebra.ts

```diff
@@ -413,6 +436,22 @@ export class Line {
     return distance;
   }
 
+  /**
+   * Find the perpendicular line from the line to a point
+   * @param point
+   */
+  public findVectorToPoint(point: Vector): Vector {
+    // https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line
```

Comment:
Would it make sense to have these comments as part of the method comments, like the comment for ClosestLine() ?